Testsuite update:

YCP ignores the integer parameter in y2log() functions
(https://github.com/yast/yast-core/blob/master/libycp/src/YCPBuiltinMisc.cc#L558)
while in Ruby it is handled properly
(https://github.com/yast/yast-ruby-bindings/blob/master/src/ruby/ycp/logger.rb#L8)

This can break tests which use Report or Service modules
(https://github.com/yast/yast-yast2/blob/master/library/modules/Report.ycp#L544)
(https://github.com/yast/yast-yast2/blob/master/library/runlevel/src/Service.ycp#L264)

In this case fixing the reporting means that some errors (originally) from
Service surface even though testedfiles has only NtpClient

diff --git a/agents/Makefile.am b/agents/Makefile.am
index 3b65eee..e69de29 100644
--- a/agents/Makefile.am
+++ b/agents/Makefile.am
@@ -1,9 +0,0 @@
-#
-# Makefile.am for devtools/devtools/skeletons/config/agents
-#
-
-scrconf_DATA =		\
-	etc_ntp.scr	\
-	cfg_ntp.scr
-
-EXTRA_DIST = $(scrconf_DATA)
diff --git a/data/Makefile.am b/data/Makefile.am
index 2178917..e69de29 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -1,9 +0,0 @@
-#
-# Makefile.am for ntp-client/data
-#
-
-ydata_DATA = \
-	ntp_servers.ycp
-
-
-EXTRA_DIST = $(ydata_DATA)
diff --git a/testsuite/Makefile.am b/testsuite/Makefile.am
index d9b12b0..e85fbc8 100644
--- a/testsuite/Makefile.am
+++ b/testsuite/Makefile.am
@@ -5,7 +5,7 @@
 #
 
 AUTOMAKE_OPTIONS = dejagnu
-EXTRA_DIST = $(wildcard tests/*.out) $(wildcard tests/*.err) $(wildcard tests/*.ycp) $(wildcard tests/*.yh)
+EXTRA_DIST = $(wildcard tests/*.out) $(wildcard tests/*.err) $(wildcard tests/*.rb)
 
 testsuite_prepare = @ydatadir@/testsuite/Makefile.testsuite
 
diff --git a/testsuite/tests/Write.out b/testsuite/tests/Write.out
index 76a6dd4..f6f0e4c 100644
--- a/testsuite/tests/Write.out
+++ b/testsuite/tests/Write.out
@@ -7,5 +7,7 @@ Write	.sysconfig.network.config nil true
 Execute	.target.bash "/sbin/netconfig update -m ntp" 0
 Write	.sysconfig.ntp.NTPD_RUN_CHROOTED "yes" true
 Write	.sysconfig.ntp nil true
+Log	Unable to disable service ntp
+Log	Cannot adjust 'NTP' service.
 Execute	.target.bash "test -e /etc/cron.d/novell.ntp-synchronize  && rm /etc/cron.d/novell.ntp-synchronize;" 0
 Return	true
diff --git a/yast2-ntp-client.spec.in b/yast2-ntp-client.spec.in
index 4b69c82..1d4089e 100644
--- a/yast2-ntp-client.spec.in
+++ b/yast2-ntp-client.spec.in
@@ -17,6 +17,8 @@ Requires:       yast2-country-data
 
 BuildArch:      noarch
 
+Requires:       yast2-ruby-bindings >= 1.0.0
+
 Summary:        YaST2 - NTP Client Configuration
 
 %description
@@ -34,11 +36,11 @@ This package contains the YaST2 component for NTP client configuration.
 %defattr(-,root,root)
 %dir @yncludedir@/ntp-client
 @yncludedir@/ntp-client/*
-@clientdir@/ntp-client.ycp
-@clientdir@/ntp-client_*.ycp
+@clientdir@/ntp-client.rb
+@clientdir@/ntp-client_*.rb
 @scrconfdir@/cfg_ntp.scr
 @scrconfdir@/etc_ntp.scr
-@moduledir@/*.y*
+@moduledir@/*.rb
 @desktopdir@/ntp-client.desktop
 @ydatadir@/ntp_servers.ycp
 %doc @docdir@
